fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingThrottle.java

```diff
@@ -84,15 +84,13 @@ public class IndexingThrottle {
     }
 
     public <R> CompletableFuture<R> buildCommitRetryAsync(@Nonnull BiFunction<FDBRecordStore, AtomicLong, CompletableFuture<R>> buildFunction,
```

Comment:
After deleting indexingByRecord, we may be able to simplify the code a bit by converting the <R> to a Boolean.